--- gnuplot-4.0.0/gnuplot-mode.0.6.0/dotemacs.pix	1999-01-01 01:16:14.000000000 +0100
+++ gnuplot-4.0.0/gnuplot-mode.0.6.0/dotemacs	2005-05-17 14:24:03.674092512 +0200
@@ -15,7 +15,7 @@
 
 ;; This line binds the function-9 key so that it opens a buffer into
 ;; gnuplot mode 
-  (global-set-key [(f9)] 'gnuplot-make-buffer)
+;  (global-set-key [(f9)] 'gnuplot-make-buffer)
 
 ;; end of line for gnuplot-mode
 ;;--------------------------------------------------------------------
